test/visual/default.html

```diff
@@ -33,7 +33,7 @@
 
   <script type="module">
     (async () => {
-      const theme = window.location.search.replace(/.*theme=(\w+).*/, '$1') || 'lumo';
+      const theme = window.location.search.replace(/.*theme=(\w+).*/, '$1') || 'material';
```

Comment:
```suggestion
      const theme = window.location.search.replace(/.*theme=(\w+).*/, '$1') || 'lumo';
```
Let's not change the defaults. User ?theme=material in the URL if you want to test with material. Full link: http://localhost:8000/test/visual/default.html?theme=material

Reply:
Yikes, my bad. Forgot to undo that part.